my-frst-react-app/src/hello.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+
+export default class Hello extends React.Component{
+
+render(){
+   
+    return(
+            <h1>Hello {this.props.name}!</h1>
+        );
+    
+     }
+    
+}
```

Comment:
Filename should always equal the class name, including casing. So this should be `Hello.js`